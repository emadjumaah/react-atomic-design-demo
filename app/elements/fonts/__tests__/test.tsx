import React from 'react';
import { render } from '@testing-library/react';
import {
  PageTitle,
  SectionTitle,
  SubsectionTitle,
  PrimaryText,
  DescriptionText,
} from 'elements';

describe('Fonts', () => {
  it('Renders PageTitle', () => {
    const { container } = render(<PageTitle>Page Title</PageTitle>);
    expect(container).toMatchSnapshot();
  });
  it('Renders SectionTitle', () => {
    const { container } = render(<SectionTitle>Section Title</SectionTitle>);
    expect(container).toMatchSnapshot();
  });
  it('Renders SubsectionTitle', () => {
    const { container } = render(
      <SubsectionTitle>Subsection Title</SubsectionTitle>,
    );
    expect(container).toMatchSnapshot();
  });
  it('Renders PrimaryText', () => {
    const { container } = render(<PrimaryText>Primary Text</PrimaryText>);
    expect(container).toMatchSnapshot();
  });
  it('Renders DescriptionText', () => {
    const { container } = render(
      <DescriptionText>Description Text</DescriptionText>,
    );
    expect(container).toMatchSnapshot();
  });
});
