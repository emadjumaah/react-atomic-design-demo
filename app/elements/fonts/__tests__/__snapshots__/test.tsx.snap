// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fonts Renders DescriptionText 1`] = `
<div>
  <p
    class="undefined description"
  >
    Description Text
  </p>
</div>
`;

exports[`Fonts Renders PageTitle 1`] = `
<div>
  <h1>
    Page Title
  </h1>
</div>
`;

exports[`Fonts Renders PrimaryText 1`] = `
<div>
  <p>
    Primary Text
  </p>
</div>
`;

exports[`Fonts Renders SectionTitle 1`] = `
<div>
  <h2>
    Section Title
  </h2>
</div>
`;

exports[`Fonts Renders SubsectionTitle 1`] = `
<div>
  <h3>
    Subsection Title
  </h3>
</div>
`;
